_posts/courses/earth-analytics-python/08-multispectral-remote-sensing-fire/in-class/2017-03-01-fire01-difference-normalized-burn-ratio-veg-indices.md

```diff
@@ -190,6 +190,12 @@ should you use to calculate NBR using MODIS?
 
 
 
+{:.output}
```

Comment:
@nkorinek can you have a look at this file?  2017-03-01-fire01-difference-normalized-burn-ratio-veg-indices.md  essentially it is printing out the homework data link. can you please ensure that output is hidden?